tests/tokenization/test_tokenization_utils.py

```diff
@@ -296,3 +304,13 @@ def test_len_tokenizer(self):
                 self.assertEqual(len(tokenizer), tokenizer.vocab_size + 1)
                 self.assertEqual(len(tokenizer.added_tokens_decoder), added_tokens_size + 1)
                 self.assertEqual(len(tokenizer.added_tokens_encoder), added_tokens_size + 1)
+
+    @require_sentencepiece
+    def test_sentencepiece_cohabitation(self):
+        from sentencepiece import sentencepiece_model_pb2 as _original_protobuf  # noqa: F401
```

Comment:
when we do this, we are importing `sentencepiece.sentencepiece_model_pb2` and it will persists druing the whole (remaining) life time of the python test process.

I would probably avoid this, and run this test in a separate test

Reply:
are you suggesting to move this test to its own file?

Reply:
sorry, typo, I mean in a separate python subprocess.

But we can wait for a core maintainer's review.

Reply:
OK! Who is the core maintainer? Can we tag him?

Reply:
I already ping Arthur